docs/installation_and_basic_usage.ipynb

```diff
@@ -2,294 +2,125 @@
     "\n",
-    "To use a package, first install `stactools` then the package. `stactools` can be installed with `pip`."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "0f81ebc5-62ce-4884-b7ee-7533661787a2",
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "%pip install stactools"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "0959bd18-aa99-4318-a7c6-e0f39f8ba081",
-   "metadata": {},
-   "source": [
-    "Check that the `stac` CLI tool is installed"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 59,
-   "id": "3d574e02-274a-4aba-b464-11e71b5b9036",
-   "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "Usage: stac [OPTIONS] COMMAND [ARGS]...\n",
-      "\n",
-      "Options:\n",
-      "  -v, --verbose  Use verbose mode\n",
-      "  -q, --quiet    Use quiet mode (no output)\n",
-      "  --help         Show this message and exit.\n",
-      "\n",
-      "Commands:\n",
-      "  copy            Copy a STAC Catalog\n",
-      "  describe        Prints out a list of all catalogs, collections and items in\n",
-      "                  this STAC.\n",
-      "\n",
-      "  info            Display info about a static STAC catalog.\n",
-      "  layout          Reformat the layout of a STAC based on templating.\n",
-      "  merge           Merge items from one STAC into another.\n",
-      "  move-assets     Move or copy assets in a STAC to the Item locations.\n",
-      "  nrcanlandcover  Commands for working with Natural Resources Canada Land\n",
-      "                  Cover data\n",
-      "\n",
-      "  sentinel2       Commands for working with sentinel2 data\n",
-      "  version         Display version info.\n"
-     ]
-    }
+    "Stactools and its dataset packages can be accessed from the CLI or from within normal Python code. This notebook provides examples of both."
    ],
-   "source": [
-    "!stac"
-   ]
+   "metadata": {}
   },
   {
    "cell_type": "markdown",
-   "id": "b99ce54c-0c58-45d0-8528-fad272b4210c",
-   "metadata": {},
    "source": [
-    "Notice the Commands available. In the next step we will add a dataset to work with"
-   ]
+    "## 1. Using this notebook\n",
+    "\n",
+    "The easiest way to use this notebook is to run it through `scripts/notebook`. This will create a Docker container already set up with the prerequisites and open the stactools dataset folder. Navigate to `docs/` to find this notebook.\n",
+    "\n",
+    "If you wish to use this notebook outside of the container then please install the prerequisites using:\n",
+    "\n",
+    "`pip install stac stactools-<name of this package>`"
```

Comment:
If we want strings like this to be replaced during the `scripts/rename` we can change it to `pip install stactools-ephemeral`